x/merkledb/trieview.go

```diff
@@ -789,14 +773,6 @@ func (t *trieView) getValue(key path, lock bool) ([]byte, error) {
 	return value, nil
 }
 
-// Insert will upsert the key/value pair into the trie.
-func (t *trieView) Insert(_ context.Context, key []byte, value []byte) error {
-	t.lock.Lock()
-	defer t.lock.Unlock()
-
-	return t.insert(key, value)
-}
-
 // Assumes [t.lock] is held.
 // Assumes [t.validityTrackingLock] isn't held.
 func (t *trieView) insert(key []byte, value []byte) error {
```

Comment:
Can we add a comment to clarify the difference between `insert` and `insertIntoTrie`?